joomla/joomla-cms
# Redirects - move Filters from left to [Search Tools]

This PR moves the Filter option of the **Redirect Manager: Links** to the [Search Tools] button.
(See Issue #6941 regarding the inconsistency with Filter options).
## Test instructions

In back-end > Components > Redirects
See the filter "- Select Status-" on the left
Also notice that the Expired URL column is too small.

![screen shot 2015-05-15 at 08 46 30](http://issues.joomla.org/uploads/1/f6b195be88171b43f16976bc973b629c.png)

Compare with the other filters, e.g. in **Category Manager**: 
In back-end > Content > Categories
### This PR changes moves the Filter

from the left to the middle column and will be visible when you click on the [Search Tools] button.
This PR also fixes the **column width** of the **Expired URL** column (now the columns Expired URL, New URL, and Referring Page are all 30%) so that it looks better on a smaller screen.

![screen shot 2015-05-15 at 08 46 30](http://issues.joomla.org/uploads/1/29d83a28b3f68d79f5bc1a167b11c29c.png)
